services/crypto_get_info.proto

```diff
@@ -141,7 +141,7 @@ message CryptoGetInfoResponse {
         /**
          * All tokens related to this account
          */
-        repeated TokenRelationship tokenRelationships = 15;
+        repeated TokenRelationship tokenRelationships = 15 [deprecated = true];
```

Comment:
This field is also used in `get_account_details.proto`. Can be deprecated there as well